include/solvers/postprocessing_cfd.h

```diff
@@ -40,6 +40,39 @@
 #  include <core/boundary_conditions.h>
 #  include <core/parameters.h>
 
+/**
+ * @brief Calculate the pressure drop between two boundaries
+ * @return Pressure drop of the flow between two boundaries of the domain
+ *
+ * @param dof_handler The dof_handler used for the calculation
+ *
+ * @param evaluation_point The solution for which the CFL is calculated. The velocity field is assumed to be the first field.
+ *
+ * @param mpi_communicator The mpi communicator. It is used to reduce the CFL calculation.
```

Comment:
You can actually get the communicator straight out of the dof_handler : see https://www.dealii.org/current/doxygen/deal.II/classDoFHandler.html
This way you could remove this extra argument. It could also be removed from the other functions and make the API cleaner :)